libs/components/config/src/lib/config-remote-modules.ts

```diff
@@ -1,18 +1,43 @@
 import { SkyuxConfigRemoteContainerReference } from './config-remote-container-reference';
 
 /**
- * Information about remote modules referenced by this application.
+ * Information about remote modules exposed and/or referenced by this application.
  */
 export interface SkyuxConfigRemoteModules {
   /**
-   * A collection of remote modules keyed by the remote container name.
+   * Whether this application can load remote modules without validating them
+   * against the list of `referenced` modules. Disabling this validation could
+   * lead to unpredictable behavior and should not be used in most cases.
+   */
+  allowDynamicLoading?: boolean;
+  /**
+   * Information about remote modules exposed by this application.
+   */
+  public?: {
```

Comment:
Adding missing properties to the `@skyux/config` version (these are already represented within the internal version).